App/Sagas/TextileSagas.js

```diff
@@ -124,6 +125,7 @@ export function * createNode ({path}) {
     const debugLevel = (__DEV__ ? "DEBUG" : "INFO")
     const createNodeSuccess = yield call(IPFS.createNodeWithDataDir, path, API_URL, debugLevel)
     if (createNodeSuccess) {
+      yield call(IPFS.updateThread, Config.ALL_THREAD_MNEMONIC, Config.ALL_THREAD_NAME)
```

Comment:
Should we be checking the return value here? If it fails, it should run the `else` below. So just to be safe, something like: move the call to `updateThread` out of this `if` and then update the `if` to `if (createNodeSuccess && updateThreadSuccess)`

Reply:
cool, fixed